### PR TITLE
logging: support `ms` duration format and add docs

### PR DESCRIPTION
**Pull request by mohammed90**
The issue caddyserver/transform-encoder#42 made the gap clear. I decided to add a few lines of docs.

### CODE REVIEW

File: modules/logging/encoders.go [hunk 1]
```diff
@@ -266,6 +275,8 @@ func (lec *LogEncoderConfig) ZapcoreEncoderConfig() zapcore.EncoderConfig {
 		durFormatter = zapcore.NanosDurationEncoder
 	case "string":
 		durFormatter = zapcore.StringDurationEncoder
+	case "ms":
```
Comment (francislavoie) [inline modules/logging/encoders.go:278]: The options are kinda misaligned I think, nano and ms aren't the same, should be nano and milli for example. But we could support all of them. So ns/nano, s/second, ms/milli etc

Comment (mohammed90) [reply modules/logging/encoders.go]: Better?

Comment (francislavoie) [reply modules/logging/encoders.go]: Yep, thanks!

Thinking we should probably add docblock comments to the rest of the fields too, but this is good enough for now.

**Review by francislavoie — APPROVED** [review]

**Comment by mholt** [comment]
Cool, thanks Mohammed :)

**Pull request merged**